web/quiz_views.py

```
@@ -17,7 +19,82 @@
+    exam_type = "course" if not session else "session"
+
+    if course is None:
+        return HttpResponseForbidden("A course or session id is required to create an exam.")
+
+    # Handle request
+    if request.method == "POST":
+        form = QuizForm(request.POST)
+        if form.is_valid():
+            quiz = form.save(commit=False)
+            quiz.creator = request.user
+            # Ensure global uniqueness – retry until an unused code is found
```

Ruff (0.8.2)

Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)